app/views/bets/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+  <% if current_user %>
+    <h2> Creating a new bet </h2>
+    <form action="<% create_bet_path %>" method="post" id="create-bet-form">
+      <div>
+        <input type="hidden" value="<%= form_authenticity_token %>" name="authenticity_token"/>
+        <input type="hidden" value="<%= current_user.id %>" name="creator_id"/>
+        <input type="hidden" value="1" name="status"/>
+      </div>
+
+      <div class="form-group">
+        <label for="bet-text">Enter bet text:</label>
+        <input type="text" class="form-control" id="bet-text" name="bet">
+      </div>
+
+      <div class="form-group">
+        <label for="bet-expdate">Enter bet expiration date (placing your bets is not allowed after this date):</label>
+        <input type="text" class="form-control jquery-datepicker" name="expires_at">
+        </input>
+      </div>
+
+      <div class="form-group">
+        <button type="submit" class="btn btn-primary">Create bet</button>
+      </div>
+    </form>
+
+  <% else %>
+    <h2>You are not logged in </h2>
+<% end %>
```

Comment:
Good catch, thanks! 
Just feels like doing it manually gives me a bit more control over the style, but I might be mistaken here. Is it considered a better idea to use form_for in situations like that?